onboarding.md

```diff
@@ -0,0 +1,43 @@
+# Onboarding
+
+This document covers steps needed to onboard new Steering Committee
+members and off-board emeritus members.
+
+- [ ] All new members must complete the [Inclusive Leadership Training].
+
+- [ ] Update the following files in the respective repos:
+  - [ ] [kubernetes/steering]: `OWNERS_ALIASES` and `README.md`
+  - [ ] [kubernetes/community]: `sigs.yaml` and auto-generated content
+  - [ ] [kubernetes/org]: `OWNERS_ALIASES` and the `steering-committee`
+    GitHub team
+  - [ ] [kubernetes/funding]: `OWNERS_ALIASES`
+  - [ ] [kubernetes/kubernetes-template-project]: `OWNERS_ALIASES`
+
+- [ ] Add new members with Owner access to the `steering-private`
+  mailing list and remove emeritus members.
+
+- [ ] Grant new members Owner access to the public steering mailing
```

Comment:
I think this is still left for us to do

Reply:
Done as a part of https://github.com/kubernetes/k8s.io/pull/419